src/Helpers/Validation.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * Helpers for validation
+ *
+ * @package EightshiftForms\Helpers
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Helpers;
+
+/**
+ * Helpers for validation
+ */
+class Validation
+{
+
+	/**
+	 * Maximum file size expressed in B.
+	 *
+	 * @var integer
+	 */
+	public const MAX_FILE_SIZE = 5 * 1024 * 1024;
+
+	/**
+	 * Minimum file size expressed in B.
+	 *
+	 * @var integer
+	 */
+	public const MIN_FILE_SIZE = 1;
+
+	/**
+	 * Check if string contains url
+	 *
+	 * @param string $string String to check.
+	 *
+	 * @return boolean
+	 */
+	public static function containsUrl(string $string): bool
+	{
+		return (bool) preg_match('/(http|ftp|mailto)/', $string);
+	}
+
+	/**
+	 * Validate File Minimum size.
+	 *
+	 * @param integer $fileSize File size value.
+	 *
+	 * @return boolean
+	 */
+	public static function isFileMaxSizeValid(int $fileSize): bool
+	{
+		return $fileSize <= static::MAX_FILE_SIZE ? true : false;
```

Comment:
You don't need a ternary expression here: `return $fileSize <= static::MAX_FILE_SIZE` would be enough. See https://3v4l.org/t02XW

For all comparison operators apart from the spaceship operator, the result of a comparison is [always a bool](https://3v4l.org/t02XW), so this seems just a bit redundant. Not really a (performance) issue as PHP should automatically optimize this, though.